app/admin/news_article.rb

```diff
@@ -0,0 +1,53 @@
+if defined?(ActiveAdmin)
+  ActiveAdmin.register News::Article, as: 'News' do
+    config.batch_actions = false
+
+    index do
+      selectable_column
+      column :id
+      column :title
+      column :annonce
+      actions
+    end
+
+    show do
+      attributes_table do
+        row :id
+        row :title
+        row (:text) { |article| raw(article.text) }
+        row :text
+      end
+    end
+
+    form do |f|
+      inputs do
+        input :title
+        input :annonce
+        input :text, input_html: { class: 'ckeditor' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.